hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyLocationInfoGroup.java

```diff
@@ -58,11 +58,22 @@ public long getVersion() {
     return locationList;
   }
 
+  public KeyLocationList getCompactProtobuf() {
+    return KeyLocationList.newBuilder()
+        .setVersion(version)
+        .addAllKeyLocations(
+            locationList.stream().map(
+                OmKeyLocationInfo::getCompactProtobuf)
+                .collect(Collectors.toList()))
+        .build();
+  }
+
```

Comment:
Can we avoid duplication by extracting to a helper method that takes `OmKeyLocationInfo` conversion function (`OmKeyLocationInfo::getCompactProtobuf` or `OmKeyLocationInfo::getProtobuf`)?